tests/unit/test_ods.py

```diff
@@ -0,0 +1,214 @@
+import mock
+import unittest
+import functools
+
+import app.helpers.ods as ods
+
+from hypothesis import strategies as st
+from hypothesis import given, example, assume
+
+po = functools.partial(mock.patch.object, autospec=True)
```

Comment:
This feels like a better approach than repeating it in every test method. Is there a reason view tests aren't doing the same?

I'd also go a step further and move this to the test helpers, but with a more descriptive name.